Sources/SmileID/Classes/Networking/RestServiceClient.swift

```diff
@@ -1,7 +1,8 @@
-import Foundation
 import Combine
+import Foundation
 
 protocol RestServiceClient {
     func send<T: Decodable>(request: RestRequest) -> AnyPublisher<T, Error>
+    func multipart(request: RestRequest) -> AnyPublisher<SmartSelfieResponse, Error>
```

Comment:
Is it possible to make this generic? multipart should not be tied directly to SmartSelfieResponse

Reply:
I actually tried that first, and I ran into issues

I wanted to make it look like the send protocol

Reply:
What kind of issues? why can't we do something like `func multipart<T: Decodable>(request: RestRequest) -> AnyPublisher<T, Error>`